lib/faster_path/optional/monkeypatches.rb

```diff
@@ -3,7 +3,11 @@ def self.sledgehammer_everything!
     ::File.class_eval do
       def basename(pth)
         FasterPath.basename(pth)
-      end 
+      end
+
+      def dirname(pth)
```

Comment:
Should be `self.dirname`. As it is, this doesn't override the existing method but adds an instance method instead.